source/FreeRTOS_TCP_WIN.c

```diff
@@ -1901,18 +1901,33 @@
                                                      const TCPSegment_t * pxSegment )
         {
             int32_t mS = ( int32_t ) ulTimerGetAge( &( pxSegment->xTransmitTimer ) );
+            int32_t lSum = 0;
+            int32_t weight = 0;
+            int32_t divisor = 0;
```

Comment:
NIT: Naming can be as per the usual FreeRTOS coding guidelines. For int32_t it can be like lWeight, lDivisior, etc. 
https://freertos.org/Documentation/02-Kernel/06-Coding-guidelines/02-FreeRTOS-Coding-Standard-and-Style-Guide

Reply:
Updated. Thanks!